task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue

```diff
@@ -371,10 +371,11 @@ export default {
         } else if (!this.taskTitleValid){
           this.$root.$emit('show-alert', {type: 'error',message: this.$t('alert.error.title.length')});
         } else if (this.task.id != null) {
-          this.$root.$emit('task-updated', this.task);
           this.$taskDrawerApi.updateTask(this.task.id, this.task).then(() => {
             this.taskTitle_ = this.task.title;
             this.oldTask.title = this.task.title;
+            this.$root.$emit('update-task-list', this.task);
+            this.$root.$emit('task-updated', this.task);
```

Comment:
Could we have a single event fired, that we receive from different components and update them